### DISCUSSION TITLE
Customizable Alert Schedules / Users

### DISCUSSION THREAD

**Discussion by lazerusrm** [amidaware/tacticalrmm#1420] [Ideas]
So far really enjoying the system. Love the app and working good on linux / arm servers.

Id love to see the following:

Linux scripts (id attempt to write my own if i had some direction!) E.G. CPU Temp, storage monitoring, storage smart monitoring, network load monitoring etc.)

agent installer via curl script for linux headless systems.. 

Ex: Create & deploy agent,  copy and paste command to server via terminal 

Curl -fsSL https://api.rmm.yoursite.com/ID/5aed-144sc.sh | sh

Currently having to scp the file directly to the server with different means, and this takes quite a few extra steps. (100+ servers)

Ability to change when and who gets alerts (Mainly SMS)

Example:

Team A: M-F 8AM-5PM
Team B: M-F 5PM-8AM
Cust. 17 - Site Specific

etc.

**Comment by silversword411** [comment]
Join discord, chat in #scripts . Happy to work with you on getting more Linux scripts in the community library so you can just run them from TRMM

**Reply by lazerusrm** [reply]
Thanks! Joined.
